invenio_app_rdm/records_ui/views/records.py

```diff
@@ -146,6 +147,12 @@ def record_detail(
     pid_value, record, files, media_files, is_preview=False, include_deleted=False
 ):
     """Record detail page (aka landing page)."""
+    if not request.accept_mimetypes.accept_html:
+        accept_mimetype = request.accept_mimetypes.best_match(record_serializers.keys())
```

Comment:
I think that here, we should not use the `record_serializers` hardcoded dict, but get the configured serializers via `RDM_RECORDS_SERIALIZERS` config key instead. WDYT?